### PR TITLE
Fix benchmarks template whitespaces

### PR DESCRIPTION
**Pull request by thiolliere**
As far as I see this is working

### CODE REVIEW

File: frame/system/src/weights.rs [hunk 1]
```diff
@@ -43,91 +43,80 @@ use sp_std::marker::PhantomData;
...
+	fn kill_storage(i: u32, ) -> Weight;
+	fn kill_prefix(p: u32, ) -> Weight;
 	fn suicide() -> Weight;
-
 }
 
 /// Weights for frame_system using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
-impl<T: crate::Trait> WeightInfo for SubstrateWeight<T> {
+impl<T: frame_system::Trait> WeightInfo for SubstrateWeight<T> {
```
Comment (shawntabrizi) [inline frame/system/src/weights.rs:57]:
```suggestion
impl<T: crate::Trait> WeightInfo for SubstrateWeight<T> {
```

**Review by shawntabrizi — APPROVED** [review]

**Comment by thiolliere** [comment]
bot merge

**Pull request merged**